service/src/main/java/uk/gov/hmcts/reform/fpl/model/Others.java

```diff
@@ -1,16 +1,34 @@
 package uk.gov.hmcts.reform.fpl.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.google.common.collect.ImmutableList;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
 import uk.gov.hmcts.reform.fpl.model.common.Element;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
 @Data
 @Builder
 @AllArgsConstructor
 public class Others {
     private final Other firstOther;
     private final List<Element<Other>> additionalOthers;
+
+    @JsonIgnore
```

Comment:
:( Keeping these first other / additional others is depressing. Better would be to use either JsonCreator if only one deserialisation is needed or pair serialisers / deserialisers.

I guess not for now.

Reply:
We had plans to migrate this but unfortunately never happened.